lib/rummager_finder_publisher.rb

```diff
@@ -9,8 +9,14 @@ def initialize(metadatas, logger: Logger.new(STDOUT))
 
   def call
     metadatas.each do |metadata|
-      if metadata[:file].has_key?("content_id")
+      if metadata[:file].has_key?("content_id") && !preview_only?(metadata)
```

Comment:
Should the `has_key?("content_id")` test also apply to `preview_only?` finders?

If so, I think this would be better done by nesting the preview_only test within the `has_key?("content_id")` test:

```
    if metadata[:file].has_key?("content_id")
      if preview_only?(metadata)
...
      else
....
      end
```